src/Altinn.Notifications.Core/Integrations/Consumers/PastDueOrdersConsumer.cs

```diff
@@ -17,7 +18,9 @@ public class PastDueOrdersConsumer : IHostedService
 {
     private readonly IOrderProcessingService _orderProcessingService;
     private readonly ILogger<PastDueOrdersConsumer> _logger;
+    private readonly IKafkaProducer _producer;
     private readonly KafkaSettings _settings;
+    private readonly string _pastDueOrdersTopicRetryFirst;
```

Comment:
long param name. Could we simply call it retry topic? As long as we connect it to the right value from settings, it doesn't add value to this class to have a more descriptive parameter name